src/foam/nanos/servlet/VirtualHostRoutingServlet.js

```diff
@@ -180,6 +180,7 @@ foam.CLASS({
         out.println("<link href=\\"https://fonts.googleapis.com/css?family=Roboto:100,300,400,500\\" rel=\\"stylesheet\\">");
         out.println("<link href=\\"https://fonts.googleapis.com/css?family=Lato:400,400i,700,700i,900,900i\\" rel=\\"stylesheet\\">");
         out.println("<link href=\\"https://fonts.googleapis.com/css2?family=IBM+Plex+Sans:wght@400;500;600;700&display=swap\\" rel=\\"stylesheet\\">");
```

Comment:
@mritunjay51 please remove this line

Reply:
out.println("<link href=\\"https://fonts.googleapis.com/css2?family=IBM+Plex+Sans:wght@400;500;600;700&display=swap\\" rel=\\"stylesheet\\">");

Do I have to remove only this font?

Reply:
yes

Reply:
ok, I removed that file and commit the latest change.